src/components/BackgroundDiv.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import ContrastContent from './ContrastContent';
+
+interface BakcgroundDivProps {
+  backgroundColor: string;
+  children?: ReactNode;
+}
+
+const StyledBackgroundDiv = styled(({ backgroundColor, ...rest }) => <div {...rest} />)`
+  width: 100%;
+  @media (min-width: ${({ theme }) => theme.breakpoints.values.sm + 'px'}) {
+    padding: 1rem 2rem;
+  }
+  padding: 0.5rem;
+  ${({ backgroundColor }) => `background-color: ${backgroundColor}`}
+`;
+
+const BackgroundDiv = ({ children, ...props }: BakcgroundDivProps) => (
+  <StyledBackgroundDiv {...props}>
+    <ContrastContent backgroundColor={props.backgroundColor}>{children}</ContrastContent>
```

Comment:
noen grunn til å brette ut props?
```suggestion
const BackgroundDiv = ({ children, backgroundColor }: BakcgroundDivProps) => (
  <StyledBackgroundDiv backgroundColor={backgroundColor}>
    <ContrastContent backgroundColor={backgroundColor}>{children}</ContrastContent>
```

Reply:
njei... - data-testid blir kanskje satt uansett?

Reply:
nei, data-testid blir ikke satt med mindre vi sender props videre